### ISSUE TITLE
Lage test i nais/api som fanger opp at nais-verification fungerer

### ISSUE THREAD

**Issue by mortenlj** [nais/api#68]
# Lage test i nais/api som fanger opp at nais-verification fungerer

## Beskrivelse / Essensen

Featuren nais-verification bruker nais-api til å opprette nais-verification teamet.
I praksis gjøres dette kun ved opprettelse av en ny tenant, så det er fort gjort å glemme seg når man endrer API'et og ikke legge merke til at nais-verification har brukket før vi evt. oppretter en ny tenant.

## Investeringsvilje

1 uke

## Mulig løsning i grove trekk

Trekke ut de API-kallene nais-verification gjør, og lage tester i nais-api som kjører de kallene og sjekker forventet oppførsel.
Gjerne med litt dokumentasjon om hvorfor disse testene ligger der, slik at man vet hva man skal gjøre hvis de brekker.

For bonus-poeng kunne man kanskje gjort det slik at nais-verification dytter oppdaterte queries inn i nais-api repoet ved behov.

**Comment by jhrv** [comment]
Er dette fremdeles aktuelt? 
Hva tenker @christeredvartsen @thokra-nav ?

**Comment by thokra-nav** [comment]
Ja, så lenge nais-verification ansees som en viktig komponent så er det ingenting i veien for at lignende jobb gjøres som en integrasjonstest i nais-api. 
Men det må gjøres en jobb for å få nais-api til å støtte nye service account oppsettet, så vi bør først ta en prat om hvordan vi skal løse initiell deploy

**Comment by jhrv** [comment]
Høres ut som nevnte prat bør ta plass før en evt. repitch 👍
